### DISCUSSION TITLE
Can we delay updates until X number of days since the last update?

### DISCUSSION THREAD

**Discussion by huyz** [getwud/wud#507] [Ideas]
Just as suggested in [Update containers only if there has not been an update in last X days · Issue #1847 · containrrr/watchtower](https://github.com/containrrr/watchtower/issues/1847), it would be nice to avoid being among the first to experience possible bugs from a new release for certain images.

So would it be possible to delay an update for specific images until things have been quiet for a certain number of days?

**Comment by armond-avanes** [comment]
@fmartinou This could be a very practical and helpful feature to avoid sudden failures with the latest versions. Any chance we can get this implemented in a future v8 release ?

**Comment by fmartinou** [comment]
Feature merged via #1245. It will be included in the next release. Thanks for the idea!
